src/index.html

```diff
@@ -8,6 +8,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
```

Comment:
Such structure should be fine
![image](https://user-images.githubusercontent.com/87986955/228903658-493e569d-3439-4d3e-a7b2-f93a0608850b.png)

Reply:
I grouped stars in row.